committees/tests/test_committee_detail_view.py

```diff
@@ -77,6 +83,32 @@ def test_committee_list_view(self):
         self.assertQuerysetEqual(res.context['topics'],
                                  ["<Topic: hello>"])
 
+    @waffle_testutils.override_flag('show_committee_topics', active=False)
+    def test_committee_list_view_topics_are_hidden_when_flag_inactive(self):
+        res = self.client.get(reverse('committee-list'))
+        self.assertEqual(res.status_code, 200)
+        with self.assertRaises(KeyError, msg="'topics' key should not appear"
```

Comment:
Better to encapsulate this method as a "verify method" I had to read three times to understand the tests..